### PR TITLE
Update color tokens

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #63 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

* Update design token version and all other dependencies
* Refactor color tokens by isolating the color tokens and creating tier 3 tokens for repeatable rules
* Replace hardcoded size values with tokens

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

## Verification
- [ ] Background colors look correct visually
- [ ] Background tier 2 token matches figma
- [ ] All layout styles look correct visually

Figma: https://www.figma.com/design/tSkHw1uJfDHy0Ml0aDp7Io/branch/SWOHKZpz2K78PPL43uvp5z/Auro-color-palette-(Variables)?node-id=4230-13084&m=dev

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/color.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+/* stylelint-disable scss/dollar-variable-empty-line-before, at-rule-empty-line-before, order/properties-order, scss/selector-nest-combinators, declaration-empty-line-before, 
+   scss/at-extend-no-missing-placeholder,  declaration-no-important, selector-max-combinators, selector-max-compound-selectors, no-descending-specificity, color-function-notation */
+
+::slotted(*) {
+  color: var(--ds-auro-popover-text-color);
+}
+
+.popover {
+  background-color: var(--ds-auro-popover-background-color);
+  box-shadow: -2px 0 5px 2px rgba(0, 0, 0, .08), 0 2px 5px 1px rgba(0, 0, 0, .08);
```
Comment (jason-capsule42) [inline src/color.scss:15]: box-shadow color should be a tier 3 token.

File: src/color.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+/* stylelint-disable scss/dollar-variable-empty-line-before, at-rule-empty-line-before, order/properties-order, scss/selector-nest-combinators, declaration-empty-line-before, 
+   scss/at-extend-no-missing-placeholder,  declaration-no-important, selector-max-combinators, selector-max-compound-selectors, no-descending-specificity, color-function-notation */
+
+::slotted(*) {
+  color: var(--ds-auro-popover-text-color);
+}
+
+.popover {
+  background-color: var(--ds-auro-popover-background-color);
```
Comment (jason-capsule42) [inline src/color.scss:14]: this should be a 'container' token not a 'background' token. I know we get to make up our own tier 3 tokens here but we should follow the semantic name of the tier 2 in this sense.

File: src/color.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  box-shadow: -2px 0 5px 2px rgba(0, 0, 0, .08), 0 2px 5px 1px rgba(0, 0, 0, .08);
+}
+
+.arrow {
+  &:before {
+    background-color: var(--ds-auro-popover-background-color);
```
Comment (jason-capsule42) [inline src/color.scss:20]: Same container token name comment

File: src/tokens.scss [hunk 4]
```diff
@@ -0,0 +1,4 @@
+:host {
+  --ds-auro-popover-background-color: var(--ds-color-container-primary-default, $ds-color-container-primary-default);
+  --ds-auro-popover-text-color: var(--ds-color-text-primary-default, $ds-color-text-primary-default);
```
Comment (jason-capsule42) [inline src/tokens.scss:3]: These are missing the #{...} around the SCSS variable in order to make them compile correctly.

Comment (jordanjones243) [reply src/tokens.scss:3]: Based on already released code, we only need to use `#{...}` when reassigning the value of the tokens in `color.scss`

File: src/tokens.scss [hunk 5]
```diff
@@ -0,0 +1,6 @@
+:host {
+  --ds-auro-popover-arrow-boxshadow-color: rgb(0 0 0 / 0.08);
+  --ds-auro-popover-boxshadow-color: rgb(0 0 0 / 0.08), 0 2px 5px 1px rgb(0 0 0 / 0.08);
```
Comment (jason-capsule42) [inline src/tokens.scss:3]: Why is the color in here twice? And why the offset numbers?

Also, this should be changed to use the closest matching black opacity token in the new tokens.

File: src/auro-popover.js
Comment (jason-capsule42) [inline src/auro-popover.js:1]: Cleanup all the linting issues, please. :)

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: src/style.scss [hunk 6]
```diff
@@ -129,22 +123,24 @@ $auro-inset-directions: '';
...
-    left: -6px;
+    left: calc(-1 * var(--ds-size-200, $ds-size-200));
 
     transform: rotate(45deg);
   }
 }
 
 [data-popper-placement^='bottom'] > .arrow {
-  top: -10px;
+  // calc = -10px
```
Comment (jason-capsule42) [inline src/style.scss:138]: Remove commented out code

File: src/tokens.scss [hunk 7]
```diff
@@ -0,0 +1,6 @@
+:host {
+  --ds-auro-popover-arrow-boxshadow-color: var(--ds-color-border-divider-default, $ds-color-border-divider-default);
+  --ds-auro-popover-boxshadow-color: var(--ds-color-border-divider-default, $ds-color-border-divider-default);
```
Comment (jason-capsule42) [inline src/tokens.scss:3]: These should *always* be the same color. We can use line 3 token anywhere you are using line 2.

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**